x-pack/plugins/security_solution/common/endpoint/types/index.ts

```diff
@@ -7,6 +7,7 @@
 import { ApplicationStart } from 'kibana/public';
 import { NewPackagePolicy, PackagePolicy } from '../../../../fleet/common';
 import { ManifestSchema } from '../schema/manifest';
+import { JsonObject } from '../../../../infra/common/typed_json';
```

Comment:
Was this import from `infra` on purpose? I don't think we currently have a dependency set on it that Plugin (although, this is only importing a type)
I see other uses of `JsonObject` in security solution being imported intead from `[kibana]/src/plugins/kibana_utils/common`